source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -1118,9 +1122,14 @@ void PubSubMessageHandler::handleChannelMessageCustom(Common::Redis::RespValuePt
   ENVOY_LOG(debug, "message received on channel '{}' on shardIndex : '{}'", value->toString(),shardIndex);
 
   if (value->type() != Common::Redis::RespType::Array ||
-      value->asArray()[0].type() != Common::Redis::RespType::BulkString) {
-    ENVOY_LOG(error, "unexpected message format: '{}'", value->toString());
-    return;
+      value->asArray()[0].type() != Common::Redis::RespType::BulkString) {        
```

Comment:
Add comments to explain why we needed this hack

Reply:
Added ji